### PR TITLE
FAQ and FireFly Tutorial Updates

### PR DESCRIPTION
**Pull request by trevorsc19**
Two main changes in this PR:
1- FAQ updates. Added some FAQs related to deploying smart contracts, connecting to Metamask, and how FireFly enables multi-chain applications
2- FireFly tutorial updates. Fixed some links to swagger pages that were outdated, updated some instructions, added instructions and screenshots on using the FireFly sandbox in order to follow along some of the examples given in the tutorials

### CODE REVIEW

**Review by nguyer — CHANGES_REQUESTED** [review]
Just a couple of minor spelling things. Also a couple notes on the FAQs. Up to you, how much you want to incorporate those. Otherwise looks good, thanks!

**Comment by nguyer** [comment]
It also looks like the docs build has failed. To test this locally, you can run `make docs` to ensure all the checks pass.

**Comment by trevorsc19** [comment]
I've made changes based on the comments from Nicko. Running `make docs` fails due to some missing alt attributes and one external link issue. Fixing those now and will push again

**Review by shorsher — APPROVED** [review]
Looks good to me, but I'll let @nguyer give the final pass.

**Review by nguyer — CHANGES_REQUESTED** [review]
There's just one section that should be removed, but otherwise looks good. Thanks!

File: docs/faqs/faqs.md [hunk 1]
```diff
@@ -23,14 +23,15 @@ In order to deploy the contract on the public Ethereum network, you'll need the
...
+Kaleido has built in Smart Contract deployment and management functionality. Additionally, you can you use the [Kaleido Truffle box](https://docs.kaleido.io/developers/smart-contracts/truffle/) to help you handle the management, deployment, and testing of smart contracts. See our [documentation](https://docs.kaleido.io/kaleido-services/smart-contract-management/usage/) here to learn more about how you can deploy smart contracts to your private network hosted on Kaleido. 
 
 For additional information about Smart Contracts, please see the official [Ethereum.org documentation](https://ethereum.org/en/developers/docs/smart-contracts/)
 
 ## 🦊 Can I connect FireFly to MetaMask?
 Yes! Before you set up MetaMask you'll likely want to create some tokens that you can use to send between wallets on your FF network. Go to the tokens tab in your FireFly node's UI, create a token pool, and then mint some tokens. Once you've done this, follow the steps listed [here](https://hyperledger.github.io/firefly/tutorials/tokens/erc721.html#use-metamask) to set up MetaMask on your network.
 
+## Can I connect FireFly to an existing Fabric Network
+
```
Comment (nguyer) [inline docs/faqs/faqs.md:34]: Should this have content in it? Or should we wait to add the header until we have the content ready?

**Review by nguyer — APPROVED** [review]
Looks good! Thanks @trevorsc19

**Pull request merged**